### PR TITLE
Remove change node channel

### PR DESCRIPTION
**Pull request by al1img**

### CODE REVIEW

**Review by mlohvynenko — APPROVED** [review]

**Review by mlohvynenko — APPROVED** [review]

**Review by MykolaSolyanko — APPROVED** [review]

**Review by mlohvynenko — APPROVED** [review]

File: utils/pbconvert/pbconvert.go [hunk 1]
```diff
@@ -45,42 +68,48 @@ func InstanceFilterToPB(filter cloudprotocol.InstanceFilter) *pb.InstanceIdent {
...
+	return aostypes.InstanceIdent{
+		ServiceID: ident.GetServiceId(),
+		SubjectID: ident.GetSubjectId(),
+		Instance:  ident.GetInstance(),
 	}
+}
 
-	copy(networkParams.GetDnsServers(), params.DNSServers)
+// InstanceIdentToPB converts InstanceIdent from AOS type to protobuf.
+func InstanceIdentToPB(ident aostypes.InstanceIdent) *pbcommon.InstanceIdent {
```
Comment (mykola-kobets-epam) [inline utils/pbconvert/pbconvert.go:82]: It's not clear why some functions contain New prefix, some doesn't

File: api/cloudprotocol/unitstatus.go [hunk 2]
```diff
@@ -64,6 +69,21 @@ const (
...
+// Node attribute names.
+const (
+	NodeAttrMainNode      = "MainNode"
+	NodeAttrAosComponents = "AosComponents"
+	NodeAttrRunners       = "NodeRunners"
+)
+
+// Node attribute values.
+const (
+	AosComponentCM  = "cm"
```
Comment (mykola-kobets-epam) [inline api/cloudprotocol/unitstatus.go:81]: Maybe rename them to AosServices, to be more precise?

**Review by mykola-kobets-epam — APPROVED** [review]

File: utils/pbconvert/pbconvert.go (hunk 1 above)
Comment (al1img) [reply utils/pbconvert/pbconvert.go:82]: It was done before, I just follow the pattern. Ok, will rename.

File: api/cloudprotocol/unitstatus.go (hunk 2 above)
Comment (al1img) [reply api/cloudprotocol/unitstatus.go:81]: I'm afraid it's confusing with services we run as containers.

**Pull request merged**